source/SpinalHDL/Structuring/components_hierarchy.rst

```diff
@@ -11,7 +11,7 @@ Like in VHDL and Verilog, you can define components that can be used to build a
 
 .. code-block:: scala
 
-   class AdderCell extends Component {
+   class AdderCell() extends Component {
```

Comment:
You are probably more knowledgeable than I am, a Scala quirk I never noticed since I always put the parenthesis to denote explicitly the constructor with no parameters.
So this correction is in fact a styling change and not a bug fix, so I can remove it if you prefer.

Reply:
i think that's ok either way, let's not botther about it ^^